development/build/scripts.js

```diff
@@ -89,18 +89,15 @@ const scuttlingConfigBase = {
     extra: '',
     stateHooks: '',
     nw: '',
+    // Sentry Custom Tracing
+    document: '',
```

Comment:
Note for audit sake that `document` was already an exception in MV2, this just excludes it in both MV2 and MV3.